proto-build/src/main.rs

```diff
@@ -1,12 +1,14 @@
-// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
-
-use tonic_disable_doctest::BuilderEx;
+// Copyright 2023 TiKV Project Authors. Licensed under Apache-2.0.
 
 fn main() {
+    use tonic_disable_doctest::BuilderEx;
+
     tonic_build::configure()
         .disable_doctests_for_types([".google.api.HttpRule"])
+        .emit_rerun_if_changed(false)
```

Comment:
By the way, it's interesting to know why the auto rerun is disabled.

Reply:
> By the way, it's interesting to know why the auto rerun is disabled.

After this change, we are running codegen in a new bin instead of `build.rs`, the `rerun` hint for cargo is not working anymore. Instead, they just print to the console.